scripts/requirements-minimum.txt

```diff
@@ -0,0 +1,7 @@
+python=3.6
```

Comment:
Rename the file to `tool-versions-minimum.txt`

Reply:
done